cartridge/issues.lua

```diff
@@ -3,6 +3,11 @@ local pool = require('cartridge.pool')
 local topology = require('cartridge.topology')
 local confapplier = require('cartridge.confapplier')
 local failover = require('cartridge.failover')
+local membership = require('membership')
+local vars = require('cartridge.vars').new('cartridge.issues')
+vars:new('limits', {
+    clock_treshold_sec = 5
```

Comment:
Why such value was chosen as threshold? Why user can't configure it?

Nit: typo `treshold` -> `threshold`